book/src/libs/wasp/profile_test.md

```diff
@@ -0,0 +1,108 @@
+> You can read more about it [here](./how-to/use_labels.md).
+
+---
+
+### Background Load Schedule for the `Gun`
+
+For the RPS `Gun`, we’ll define a schedule where:
+1. During the first 10 seconds, the RPS increases by 2.5 every second.
+2. For the next 40 seconds, it remains steady at 5 RPS.
+
+```go
+epsilonSchedule := wasp.Combine(
+    wasp.Steps(1, 1, 4, 10*time.Second), // Start at 1 RPS, increment by 1 RPS in 4 steps over 10 seconds (1 increment every 2.5 seconds)
+    wasp.Plain(5, 40*time.Second))       // Hold 5 RPS for 40 seconds
+```
+
+---
+
+### Virtual User Schedule
+
+For the `VirtualUser`, the schedule will:
+1. Start with 1 user for the first 10 seconds.
+2. Add 1 user every 3 seconds for 30 seconds.
+3. Gradually reduce from 10 users to 0 over 10 seconds.
+
+```go
+thetaSchedule := wasp.Combine(
+    wasp.Plain(1, 10*time.Second),         // 1 user for the first 10 seconds
+    wasp.Steps(1, 1, 9, 30*time.Second),  // Increment by 1 user every ~3 seconds over 30 seconds
+    wasp.Steps(10, -1, 10, 10*time.Second)) // Decrement by 1 user every second over 10 seconds
+```
+
+---
+
+### Defining the Profile
+
+We’ll now define our `Profile` to combine both the `Gun` and the `VirtualUser`:
+
+```go
+_, err := wasp.NewProfile().
+    Add(wasp.NewGenerator(&wasp.Config{
+        T:          t,
+        LoadType:   wasp.VU,
+        GenName:    "Theta",
+        Schedule:   thetaSchedule,
+        VU:         NewExampleScenario(srv.URL()),
+        LokiConfig: wasp.NewEnvLokiConfig(),
+    })).
+    Add(wasp.NewGenerator(&wasp.Config{
+        T:          t,
+        LoadType:   wasp.RPS,
+        GenName:    "Epsilon",
+        Schedule:   epsilonSchedule,
+        Gun:        NewExampleHTTPGun(srv.URL()),
+        LokiConfig: wasp.NewEnvLokiConfig(),
+    })).
+    Run(true)
+```
+
+---
+
+### Conclusion
+
+And that’s it! You’ve created a complex load profile that simulates a growing number of users alongside a background load that varies over time. Notice the `.Run(true)` method, which blocks until all the `Profile`'s generators have finished.
+
+You can find the full example [here](https://github.com/smartcontractkit/chainlink-testing-framework/tree/main/wasp/examples/profiles).
+
+---
+
+> [!NOTE]  
+> The `error` returned by the `.Run(true)` method only indicates if any dashboard alerts were triggered.  
+> To check for load generation errors, call the `Errors()` method on each `Generator` in the `Profile`.
+
+> [!NOTE]  
+> Currently, it’s not possible to have a "waiting" schedule that doesn’t generate any load. To implement such logic, start one `Profile` in a goroutine and use `time.Sleep` before starting the second `Profile`. An example of this can be found [here](https://github.com/smartcontractkit/chainlink-testing-framework/tree/main/wasp/examples/profiles/node_background_load_test.go).
```

Comment:
You can use `Run(false)` to not wait for the profile, you don't need to run it inside the goroutine. I'd say we remove it for now, add a case for 0 RPS and then bring the explanation back.